.gitignore

```diff
@@ -210,5 +210,8 @@ tools/MapAtmosFixer/MapAtmosFixer/bin/*
 /_maps/**/backup/
 /_maps/templates.dm
 
-# config
-config/*
```

Comment:
We aren't going to ignore whitelist.txt admins.txt comms.txt and dbconfig.txt?

Reply:
They're ignored on the server side of things. So, changes merged in for those files are not live unless I or Cthulhu manually apply them.

Reply:
Yes but some players will prefer to keep their name in as an admin for testing

Reply:
The local user is always admin. There is no reason to touch those files during testing.

Reply:
I've never had admin as the local user what